app/build.gradle.kts

```diff
@@ -180,9 +180,9 @@ dependencies {
     implementation("androidx.biometric:biometric:1.1.0")
     implementation("androidx.webkit:webkit:1.4.0")
 
-    implementation("com.google.android.exoplayer:exoplayer-core:2.15.1")
-    implementation("com.google.android.exoplayer:exoplayer-hls:2.15.1")
-    implementation("com.google.android.exoplayer:exoplayer-ui:2.15.1")
+    implementation("com.google.android.exoplayer:exoplayer-core:r2.16.1")
```

Comment:
Not sure if the `r` should be there. According to Android Studio its just the number that is changing. Also `cronet` has an update as well. I also see some deprecations in the changelog, have you tested to make sure its working properly? 

![image](https://user-images.githubusercontent.com/1634145/149454811-797c711f-6bbd-4d1b-9ff4-098958a6c45a.png)